### PR TITLE
refactor(registration-modal): improve button colors

### PR DESCRIPTION
**Pull request by raskolnikov-rodion**
This is a proposal to change the colors of the buttons inside the registration modal:

image

This change has two purposes:
- provide better differentiation between the buttons
- grab the user's attention and highlight the action we want to encourage

This is a common UI pattern for CTAs

[Update]: following feedback by @robertdown, the "No Thanks" button will keep the `btn-secondary` class.

### CODE REVIEW

**Review by robertdown — CHANGES_REQUESTED** [review]
Good use of `btn-primary` for the button receiving the action, but leave `btn-secondary` for the cancel.

We don't have good documentation on when to use some of the other btn styles, but the general style goes "Submit buttons get primary, other actions buttons get secondary, de-emphasized action or context buttons get btn-text

File: templates/login/login_core.html.twig [hunk 1]
```diff
@@ -188,8 +188,8 @@
                         <p class="message font-italic"></p>
                     </div>
                     <div class="modal-footer">
-                        <button type="button" class="btn btn-secondary submit">{{ "Submit"|xlt }}</button>
-                        <button type="button" class="btn btn-secondary nothanks">{{ "No Thanks"|xlt }}</button>
+                        <button type="button" class="btn btn-primary submit">{{ "Submit"|xlt }}</button>
+                        <button type="button" class="btn btn-light nothanks">{{ "No Thanks"|xlt }}</button>
```
Comment (raskolnikov-rodion) [inline templates/login/login_core.html.twig:192]:
```suggestion
                        <button type="button" class="btn btn-secondary nothanks">{{ "No Thanks"|xlt }}</button>
```

File: templates/login/partials/html/product_registration_modal.html.twig [hunk 2]
```diff
@@ -20,8 +20,8 @@ Product Registration modal
                 <p class="message font-italic"></p>
             </div>
             <div class="modal-footer">
-                <button type="button" class="btn btn-secondary submit">{{ "Submit"|xlt }}</button>
-                <button type="button" class="btn btn-secondary nothanks">{{ "No Thanks"|xlt }}</button>
+                <button type="button" class="btn btn-primary submit">{{ "Submit"|xlt }}</button>
+                <button type="button" class="btn btn-light nothanks">{{ "No Thanks"|xlt }}</button>
```
Comment (raskolnikov-rodion) [inline templates/login/partials/html/product_registration_modal.html.twig:24]:
```suggestion
                <button type="button" class="btn btn-secondary nothanks">{{ "No Thanks"|xlt }}</button>
```

**Comment by robertdown** [comment]
Thanks @raskolnikov-rodion!

**Pull request merged**